test/output/export-multi-namespace/expected.js

```diff
@@ -0,0 +1,3 @@
+module.run(function(){"use strict";var module1=module;exports.a=Object.create(null);exports.b=Object.create(null);module1.watch(require("module"),{"*":function(v,n){exports.a[n]=exports.b[n]=v}},0,[exports.a,exports.b]);
```

Comment:
As a bonus this now defines the specifiers in the correct order. Before I had them reversed.